Puppetfile

```diff
@@ -2,6 +2,11 @@
 
 mod 'stahnma/epel', '1.2.2'
 
+# Modules that have been extracted from core Puppet
```

Comment:
Is this a change to Puppet?  These are no longer part of puppet core and need to be explicitly loaded to be used?

Reply:
The extraction is part of Puppet 6.0.0.

Reply:
Ok, I'm still a little confused. The way we use this control repo in gatling_puppet_load_tests we always install from a puppet agent package. Are you using this control repo in a different way? I just want to make sure I understand how this is being used if there are additional consumers.

Reply:
It's used as a submodule in the puppet repo, and we run Puppet from source there as part of the benchmark tests.

Reply:
Ah, I just saw your earlier comment about seeing errors in the benchmarks in puppet repo. I get it now.